components/mindstorms-ev3/src/main/java/com/example/ev3/EV3SharedObject.java

```diff
@@ -0,0 +1,417 @@
+//
+// Copyright (c) 2004-2009 StreamBase Systems, Inc. All rights reserved.

```

Comment:
StreamBase Systems is now TIBCO :)

Reply:
Good catch : )  Should look like this now:

//
// Copyright (c) 2010-2021 TIBCO Software Inc. All rights reserved.
//